modules/proc-providing-feedback-on-redhat-documentation.adoc

```diff
@@ -0,0 +1,46 @@
+[preface]
+[id="proc-providing-feedback-on-redhat-documentation"]
+= Providing feedback on Red Hat documentation
+
+We appreciate your feedback on our documentation.
+To provide feedback, you can highlight the text in a document and add comments.
+
+This section explains how to submit feedback.
+
+.Prerequisites
+
+* You are logged in to the Red Hat Customer Portal.
+* In the Red Hat Customer Portal, view the document in *Multi-page HTML* format.
```

Comment:
```suggestion
* Log in to the Red Hat Customer Portal.
* Select *Multi-page HTML* format for the documentation.
```

Reply:
The first bullet and second bullet use different styles. Prereqs should always be imperative voice for parallel structure. I'm willing to die on this hill. 

Again I feel like this is a little unnecessary and could be condensed into a first step in the procedure. Something like: 

"Make sure you are logged in to the Red Hat Customer Portal and viewing the document in *Multi-page HTML* format.

Reply:
I like this but not the 'for the documentation' you're in the documentation so do we really need to say that?